CHANGELOG.md

```diff
@@ -9,17 +9,18 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - `mypy` for search space and objectives
 - Class hierarchy for objectives
 - Deserialization is now also possible from optional class name abbreviations
-- `Kernel`, `MaternKernel`, `AdditiveKernel`, `ProductKernel` and `ScaleKernel` 
-  classes for specifying kernels
+- `AdditiveKernel`, `CosineKernel`, `LinearKernel`, `MaternKernel`, `PeriodicKernel`, 
```

Comment:
Do we really want to have an explicit list of kernels here? Specifically since this mixes base and composite kernels.

Reply:
up for debate, I think having explicitly a list of kernels which are new in the CHANGELOG makes sense
what are arguments against it?